### ISSUE TITLE
Python 3.10 support

### ISSUE THREAD

**Issue by Jasha10** [conda-forge/pyrepl-feedstock#11]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:
There seems to be incompatibility with the newly-released Python version 3.10.
```text
$ conda install -c conda-forge python=3.10 pyrepl
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package python conflicts for:
python=3.10
pyrepl -> python_abi=3.7[build=*_cp37m] -> python[version='3.4.*|3.7.*|3.9.*|3.8.*|>=3.10,<3.11.0a0']
pyrepl -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.5,<3.6.0a0']The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.27=0
  - feature:|@/linux-64::__glibc==2.27=0
  - pyrepl -> libgcc-ng[version='>=9.3.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.27

Note that strict channel priority may have removed packages required for satisfiability.
```

<br/>
Empty Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /home/jasha10/miniconda3/envs/tmp:
#
# Name                    Version                   Build  Channel
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : tmp
    active env location : /home/jasha10/miniconda3/envs/tmp
            shell level : 3
       user config file : /home/jasha10/.condarc
 populated config files : /home/jasha10/.condarc
          conda version : 4.10.3
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __cuda=11.2=0
                          __linux=5.4.0=0
                          __glibc=2.27=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/jasha10/miniconda3  (writable)
      conda av data dir : /home/jasha10/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/jasha10/miniconda3/pkgs
                          /home/jasha10/.conda/pkgs
       envs directories : /home/jasha10/miniconda3/envs
                          /home/jasha10/.conda/envs
               platform : linux-64
             user-agent : conda/4.10.3 requests/2.25.1 CPython/3.9.5 Linux/5.4.0-87-generic ubuntu/18.04.6 glibc/2.27
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```
</details>

**Comment by kynan** [comment]
Can you please confirm this was fixed by #12 ?

**Comment by Jasha10** [comment]
Confirmed fixed!
